pemFioi/conceptViewer-1.0-mobileFirst.js

```diff
@@ -3,6 +3,7 @@ var conceptViewerStrings = {
     viewerTitle: "Aide",
     selectLanguage: "Sélectionnez un langage…",
     selectTopic: "Sélectionnez une rubrique…",
+    reloadFromExercice: "Merci de recharger la page depuis votre exercice.",
```

Comment:
We use the name "Task" everywhere, so this should be named "reloadFromTask".

Also I would say "Merci d'ouvrir cette documentation directement depuis l'exercice. Vous pouvez fermer cette fenêtre." or something like that.

Reply:
I have changed it to reloadFromTask, and also in the svn repo